src/transformers/trainer.py

```diff
@@ -1370,6 +1370,29 @@ def ipex_optimize_model(self, model, training=False, dtype=torch.float32):
 
         return model
 
+    def compare_trainer_and_checkpoint_args(self, training_args, trainer_state):
+        attributes_map = {
```

Comment:
These are some parameters I expected to be overridden via the `training_args` when resuming from a checkpoint.